src/clj/game/cards/operations.clj

```diff
@@ -235,6 +235,9 @@
       :async true
       :effect (req (let [t target
                          n (num-installed state t)]
+                     (system-msg
+                       state :corp
+                       (str "uses Biased Reporting to choose " target))
```

Comment:
This can go into a :msg entry.

Reply:
I thought so as well, but then I noticed `system-msg` has been used twice in the same card implementation, so I assumed there had to be a reason for that.

Reply:
I haven't looked at the rest of the implementation, but any time you have a "uses Card Name to X" phrase in a system message, you can move the X part to a :msg, which will create the rest of the message for you.